app/handlers.go

```diff
@@ -529,10 +529,8 @@ func (h *handler) HandleInbox(w http.ResponseWriter, r *http.Request) {
 	}
 
 	baseURL, _ := url.Parse(h.conf.BaseURL)
-	title := fmt.Sprintf("%s: main page", baseURL.Host)
-
 	acct := account(r)
-	title = fmt.Sprintf("%s: followed", baseURL.Host)
+	title := fmt.Sprintf("%s: followed", baseURL.Host)
```

Comment:
For the moment I would prefer not to change this, even though it's definitely a bug.

The work I'm doing currently in the [filtering-improvements](https://github.com/mariusor/littr.go/blob/filtering-improvements/) branch involves removing all the custom handlers in favour of a generic listing one and better middleware.